Task_2/Task_2.cpp

```diff
@@ -1,20 +1,46 @@
-﻿// Task_2.cpp : Этот файл содержит функцию "main". Здесь начинается и заканчивается выполнение программы.
-//
+﻿#include <iostream>
 
-#include <iostream>
+using namespace std;
+
+//2. ЕДИНИЦЫ. Найдите все натуральные числа до m, двоичная запись которых состоит из единиц
+
+void BinaryOne(int upTo);
 
 int main()
 {
-    std::cout << "Hello World!\n";
+	int m = 0;
+
+	cout << "Enter up to what natural number m to output numbers whose binary notation consists of ones: ";
+	cin >> m;
+	cout << endl;
+
+	if (m <= 0)
+	{
+		cout << "Error! m <= 0." << endl;
+
+		return -1;
+	}
+	else {}
+
+	BinaryOne(m);
+
+	return 0;
 }
 
-// Запуск программы: CTRL+F5 или меню "Отладка" > "Запуск без отладки"
-// Отладка программы: F5 или меню "Отладка" > "Запустить отладку"
+void BinaryOne(int upTo)
+{
+	int numbers = 3, degree = 2;
```

Comment:
Why 3 and 2?

Reply:
The first number is already written in the row. If there is 1 or 2, then 1 will be written to the list. And so I did only for the sake of beauty.
It also looks better)

Reply:
Hm.. I guess these numbers should be CONST